api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -562,7 +563,12 @@ enum IntegrationEnrollKind {
   INTEGRATION_ENROLL_KIND_MACHINE_ID_GITLAB = 15;
   INTEGRATION_ENROLL_KIND_MACHINE_ID_JENKINS = 16;
   INTEGRATION_ENROLL_KIND_MACHINE_ID_ANSIBLE = 17;
-  INTEGRATION_ENROLL_KIND_SERVICENOW = 18;
+  INTEGRATION_ENROLL_KIND_MACHINE_ID_AWS = 18;
```

Comment:
`18` is `MACHINE_ID_AWS` in prehog (both cloud and teleport), but `SERVICENOW` here. I guess this is a mistake due to us manually synchronizing `usageevents.proto` to prehog as a source of truth? Not sure how to proceed.